platforms/software/dependency-management/src/main/java/org/gradle/internal/component/resolution/failure/ResolutionFailureHandler.java

```diff
@@ -80,10 +83,16 @@
 public class ResolutionFailureHandler {
     public static final String DEFAULT_MESSAGE_PREFIX = "Review the variant matching algorithm at ";
 
+    private final InternalProblems problemsService;
```

Comment:
Looks like this variable is unused

Reply:
It's used to get the additional data builder factor.  Might be you saw something I left in here pre-rebasing, but it's used now.